### PR TITLE
add more clarity to using client key

### PR DESCRIPTION
**Pull request by thevickypedia**
merge it or drop it, just felt it would be more clearer this way

### CODE REVIEW

**Review by Gustry — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -51,7 +51,8 @@ for status in client.register(store):
 # Keep the 'store' object because it contains now the access token
 # and use it next time you want to register on the TV.
 print(store)
-# {'client_key': 'ACCESS_TOKEN_FROM_TV'}
```
Comment (supersaiyanmode) [inline README.md:54]: This line was actually supposed to be the output of print(..) statement above.

To me, saying `store = { ..}` on the line below adds to the confusion. Further, line 25 is the answer to the question how to use "store". If you feel it is lacking, feel free to send across a PR to make it less confusing. :)

As of this line, I think it adds to the confusion.

**Comment by supersaiyanmode** [comment]
I think the only way readers understand "store" is if I write a quick sample code. I will do it soon.

**Comment by thevickypedia** [comment]
> I think the only way readers understand "store" is if I write a quick sample code. I will do it soon.

I'm on it now, leaving your part untouched will add few lines to explain it better. Your suggestion really makes sense :)

File: README.md (hunk 1 above)
Comment (thevickypedia) [reply README.md:54]: 👀  on this if it helps get rid of the confusion. Else, feel free to close the PR :)

**Review by thevickypedia — COMMENTED** [review]
Modified as requested

**Comment by supersaiyanmode** [comment]
Please take a look at this PR https://github.com/supersaiyanmode/PyWebOSTV/pull/47. Let me know if this addresses your questions. I modified quite a few areas of README.md

**Comment by thevickypedia** [comment]
> Please take a look at this PR #47. Let me know if this addresses your questions. I modified quite a few areas of README.md

That's a lot more than what I was expecting :D Thanks for asking my opinion too 👍🏼

**Comment by supersaiyanmode** [comment]
Sure! Feel free to comment/suggest changes there if you think the README could be improved. I'm closing this PR for now, thanks so much for your help :)

**Pull request closed without merging**